src/main/java/racingcar/WinnerCarSelector.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WinnerCarSelector {
+
+    public List<Car> getWinnerCars(List<Car> candidateCars) {
+        int maxPosition = getMaxPosition(candidateCars);
+        return candidateCars.stream()
+                .filter(car -> car.getPosition() == maxPosition)
```

Comment:
getter를 활용해 값을 꺼내어 비교하기 보다 메시지를 보내면 어떨까요?
```suggestion
                .filter(car -> car.samePosition(maxPosition))
```

Reply:
`getPosition()` 메서드가 필요하지 않았으면 `samePosition` 메서드도 괜찮았을 것 같은데, `getPosition()`이 `maxPosition`을 구하기 위해 존재하기 때문에 `getPosition()`을 사용하도록 통일하는 게 좋아 보였어요. 

이렇게 하지 않으려면 `maxPosition`을 구하기 위해 `position` 비교 함수를 넣거나 해야 하는데, 그렇게 하는 것 보단 지금 방식이 가독성이 좋다고 생각했습니다..! 

어떤게 좋은 방법인진 고민이긴 하네요... 🤔